### PR TITLE
pkg/integration: Port tests to Windows

### PR DESCRIPTION
**Pull request by jstarks**
Just porting some simple unit tests to Windows. This came up as part of #22840.

cc @jhowardmsft

### CODE REVIEW

File: pkg/integration/utils_test.go [hunk 1]
```diff
@@ -14,30 +14,38 @@ import (
...
-	if runtime.GOOS == "windows" {
-		t.Skip("Needs porting to Windows")
+	var lsCmd *exec.Cmd
+	if runtime.GOOS != "windows" {
+		lsCmd = exec.Command("ls")
+	} else {
+		lsCmd = exec.Command("cmd", "/c", "dir")
+	}
+
+	err := lsCmd.Start()
```
Comment (cpuguy83) [inline pkg/integration/utils_test.go]: Confused by the usage here.
Why not `if err := lsCmd.Run(); err != nil { t.Fatal(err) }`

Comment (jstarks) [inline pkg/integration/utils_test.go]: That does seem simpler.

**Comment by vdemeester** [comment]
Once @cpuguy83 comment taken care of, LGTM 🐲

**Comment by jstarks** [comment]
Updated.

**Comment by thaJeztah** [comment]
LGTM

**Pull request merged**